browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -12214,42 +12202,9 @@ index d64a7dba42d94db24efa8f60ae499a76423d85e9..4364cae8ec64acde543be9c8ec18175f
      void saveBackForwardSnapshotForCurrentItem();
      void saveBackForwardSnapshotForItem(WebBackForwardListItem&);
 diff --git a/Source/WebKit/UIProcess/Cocoa/WebViewImpl.mm b/Source/WebKit/UIProcess/Cocoa/WebViewImpl.mm
-index bec2ecbd2d849ab8535a1e8f96dca647e1b62631..4d0e1e398d6b18bc24e92f7e3e83e4610083107a 100644
+index 83137953bd98b8092bb5a5ca373bf7ee15057990..4d0e1e398d6b18bc24e92f7e3e83e4610083107a 100644
 --- a/Source/WebKit/UIProcess/Cocoa/WebViewImpl.mm
 +++ b/Source/WebKit/UIProcess/Cocoa/WebViewImpl.mm
-@@ -203,7 +203,7 @@ static RetainPtr<CocoaImageAnalyzerRequest> createImageAnalyzerRequest(CGImageRe
-     return request;
- }
- 
--void WebViewImpl::requestTextRecognition(const URL& imageURL, const ShareableBitmap::Handle& imageData, const String& identifier, CompletionHandler<void(TextRecognitionResult&&)>&& completion)
```

Comment:
is this removal expected?

Reply:
This change was introduced downstream in build 1612 to fix some build error in Mac
  * [roll to 02/17/22](https://github.com/microsoft/playwright/pull/12173)
  * [downstream patch](https://github.com/dpino/WebKit/commit/e00d33c48d9386ee13eb7864d95fd7449363d23e)

The patch was upstreamed and merged in https://bugs.webkit.org/show_bug.cgi?id=236752, so now the local modifications are no longer needed.